app/frontend/styles/application.scss

```diff
@@ -61,6 +61,7 @@ $govuk-new-link-styles: true;
 @import "summary-card";
 @import "toc";
 @import "sortable_table";
+@import "metadata";
```

Comment:
I think we can move this line to `application-provider.scss`, which will avoid all the metadata css also being included in the css files for the candidate, support and api docs interfaces?

We should possibly also consider having the statistics uses its own css file rather than sharing the provider one -  but that could be a future follow-up PR.

Reply:
It's not in the provider namespace tho — it just uses default application one. Let's carve it up now!

Reply:
I'm not sure what you mean by 'namespace' here, do you mean the `frontend/styles/provider` folder? If so it could move to there?

Reply:
ah — I mean creating a new folder called `frontend/styles/publications`

Reply:
that's right